api/api/utils/image_proxy/__init__.py

```diff
@@ -123,8 +129,12 @@ def get(
                 )
                 sentry_sdk.capture_exception(exc)
         if isinstance(exc, requests.exceptions.HTTPError):
+            code = exc.response.status_code
             tallies.incr(
-                f"thumbnail_http_error:{domain}:{month}:{exc.response.status_code}:{exc.response.text}"
+                f"thumbnail_http_error:{domain}:{month}:{code}:{exc.response.text}"
+            )
+            logger.warning(
+                f"Failed to render thumbnail {upstream_url=} {code=} {media_provider=}"
             )
```

Comment:
There's no truthy check on media_provider here like in the other spot. Is that intentional?

Reply:
I think it's fine to log `None` here, but per the above comment I'll actually make it required and remove the truthy check entirely.